spec/amber/cli/commands/database_spec.cr

```diff
@@ -0,0 +1,33 @@
+require "../../../spec_helper"
+require "../../../support/helpers/cli_helper"
+require "../../../support/fixtures/cli_fixtures"
+
+include CLIHelper
+include CLIFixtures
+
+module Amber::CLI
+  describe "database" do
+    describe "sqlite" do
+      cleanup
+			scaffold_app(TESTING_APP, "-d", "sqlite")
+			db_filename = db_yml("./")["sqlite"]["database"].to_s.sub("sqlite3:", "")
+
+      it "does not create the database when `db create`" do
+        MainCommand.run ["db", "create"]
+        File.exists?(db_filename).should be_false
+      end
```

Comment:
This is expected? Do weird things happen with sqlite with a blank db?

Reply:
Sqlite file gets created with the first migration.